package.json

```
@@ -4,7 +4,7 @@
     "node": ">=10.0.0"
   },
   "scripts": {
-    "version": "git add -A packages",
+    "version": "gulp build && git add -A packages",
```

```suggestion
    "preversion": "gulp build",
    "version": "git add -A packages",
```

By putting it into a `pre` [prefixed script](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts), we don't need to rely on the Unix specific `&&` functionality but we get the net same result. IMO it's also more semantic, as it's a prerequisite.

---

As per https://github.com/lerna/lerna/tree/main/commands/version#lifecycle-scripts;

```
// preversion:  Run BEFORE bumping the package version.
// version:     Run AFTER bumping the package version, but BEFORE commit.
// postversion: Run AFTER bumping the package version, and AFTER commit.
```

We need the `gulp build` to happen after the package versions have been bumped by `lerna`, but before the commit, so I think we need to chain it as a two-part `version` script.

---

(Also, I don't use Windows regularly, but https://github.com/npm/npm/issues/4040#issuecomment-209034612 makes it sound like `&&` works on Windows?)

---

sgtm. Can leave as-is.